smk/gene_abundance.smk

```diff
@@ -453,7 +464,7 @@ rule genome_mapping_profiling:
     resources:
         sort_threads = 3,
         bedcov_threads = lambda wildcards, threads: min(10, threads),
-        mem = lambda wildcards, input, attempt: max(3*5, 10 + int(3.1*os.path.getsize(input.bwaindex[0])/1e9)) + 10*(attempt-1)
+        mem = lambda wildcards, input, attempt: max(3*5, math.ceil(5.6 + 3.1e-9*get_file_size(input.bwaindex[0]))) + 10*(attempt-1)
     conda:
```

Comment:
Since we are changing this line anyway, I think that for cohorts of usual size the `max()` would always set the initial mem to `15GB`, as the MAG collection BWA index is quite large. So maybe the `3*5` should be set more realistically.

Reply:
True, it is a remnant from using `5GB` per thread for `samtools sort`. This can be reduced!